### ISSUE TITLE
Docker Windows 11 Containers

### ISSUE THREAD

**Issue by rianblue141** [eddex/aseprite-windows-docker-build#26]
Should probably be noted somewhere that Docker won't build Windows containers on Windows 11 home or education editions, you need Windows 11 pro.

**Comment by eddex** [comment]
Correct, this is why the README file contains a [Supported Windows Editions](https://github.com/eddex/aseprite-windows-docker-build?tab=readme-ov-file#supported-windows-editions) section that says that Windows 10/11 Home is not supported.

The education version is not mentioned in there. I can add this. 👍

**Comment by VencedorLOL** [comment]
Isn't there any way of running this on Windows home?

**Comment by eddex** [comment]
The [official Docker documentation](https://docs.docker.com/desktop/setup/install/windows-install/) says:
> To run Windows containers, you need Windows 10 or Windows 11 Professional or Enterprise edition. Windows Home or Education editions only allow you to run Linux containers.

So unfortunately I don't think it is possible.
